keyboards/amptrics/0422/keymaps/default/config.h

```diff
@@ -0,0 +1,4 @@
+// Dynamic keymap
+#define DYNAMIC_KEYMAP_LAYER_COUNT 4
+#define DYNAMIC_KEYMAP_MACRO_COUNT 36
+#define STM32_FLASH_SECTORS_PER_BANK 64
```

Comment:
Default behaviours + license header.
```suggestion
// Copyright 2025 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
#pragma once

#define DYNAMIC_KEYMAP_MACRO_COUNT 36
```